cesium_app/plot.py

```diff
@@ -11,6 +11,21 @@
 
 
 def feature_scatterplot(fset_path, features_to_plot):
+    """Create scatter plot of feature set.
+
+    Parameters
+    ----------
+    fset_path : str
+        Path to feature set to be plotted.
+    features_to_plot : list of str
+        List of feature names to be plotted.
+
+    Returns
+    -------
+    tuple
```

Comment:
Instead of tuple, document these either as two separate outputs, or use (fig.data, fig.layout) here